src/main/java/io/pravega/perf/PravegaWriterWorker.java

```diff
@@ -16,25 +16,32 @@
 import io.pravega.client.ClientFactory;
 import io.pravega.client.stream.impl.ByteArraySerializer;
 import io.pravega.client.stream.EventWriterConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * Class for Pravega writer/producer.
  */
 public class PravegaWriterWorker extends WriterWorker {
+    private static Logger log = LoggerFactory.getLogger(PravegaWriterWorker.class);
```

Comment:
This change may not be needed.

Reply:
I believe that you are right that it is not needed now. I needed some extra logging during development so I added it but have since removed the log.info lines. It is useful to have for any future logging needs and doesn't significantly add to runtime performance.